content/pubs/pubs.bib

```diff
@@ -1,3 +1,17 @@
+@article{sfn2018chapter,
+  title={Graph Classification Using Signal-Subgraphs: Applications in Statistical Connectomics},
+  author={Vogelstein, Joshua T},
+  journal={SfN 2018 course book},
+  year={2018},
+  url={https://neurodata.io/talks/sfn_2018_coursebook.pdf},
+}
+@article{sfn2017chapter,
+  title={What is Connectome Coding?},
```

Comment:
I believe you have the articles mixed up between 2017/2018.

Reply:
Thanks! Fixed